pic-sure-hpds-ui/httpd/httpd-vhosts.conf

```diff
@@ -48,13 +48,15 @@ Mutex  "file:${HTTPD_PREFIX}/logs/ssl_mutex"
     RewriteEngine On
     ProxyPreserveHost On
 
-    RewriteRule ^/picsure/(.*)$ "https://wildfly:8080/pic-sure-api-2/PICSURE/$1" [P]
+    RewriteRule ^/picsure/(.*)$ "http://wildfly:8080/pic-sure-api-2/PICSURE/$1" [P]
+    RewriteRule ^/psama/(.*)$ "http://wildfly:8080/pic-sure-auth-services/auth/$1" [P]
 
     RewriteCond %{DOCUMENT_ROOT}/%{REQUEST_FILENAME} !-f
     RewriteCond %{DOCUMENT_ROOT}/%{REQUEST_FILENAME} !-d
 
-    RewriteRule /picsureui/(.*) /picsureui/index.html [C]
+    RewriteRule /picsureui/(.*) /picsureui/index_03272020.html [C]
     RewriteRule ^/static/(.*)$ /static/$1 [L]
+    RewriteRule ^/psamaui/(.*)$ /picsureui/index_03272020.html [C]
```

Comment:
This file is really just an example, it is overridden by all project specific repos